content/response.md

```diff
@@ -1,19 +1,28 @@
+Dear Prof. Ouellette,
+
+Thank you for providing us the opportunity to revise our manuscript.
+We are grateful for the reviewers' comments and suggestions on how to improve our manuscript and have modified the text accordingly.
+
 > ## Reviewer 1
 >
-> The authors have specified the importance of using Deep Learning in Biological data and on what are the aspects they have to focus when the researchers are using the Deep Learning. 
-> Even though they have mentioned ten different areas they have to focus the authors did not mentioned enough suitable examples and some common practices for some areas. Further it is important to mention some of the difficulties and limitations they have to face when they are using these methods. 
-
+> The authors have specified the importance of using Deep Learning in Biological data and on what are the aspects they have to focus when the researchers are using the Deep Learning.
+> Even though they have mentioned ten different areas they have to focus the authors did not mentioned enough suitable examples and some common practices for some areas. Further it is important to mention some of the difficulties and limitations they have to face when they are using these methods.
 
 > Specially need to identify what sort of the data need to be there and how to react when there is not enough data to carry out Deep Learning technique. In the paper some area not contain precise information with relevant references to support the statements. Hence these areas need to improve it further.
 
 We appreciate the constructive feedback and added additional references and clarifications where we thought it was needed.
-For instance, we provided more references and an example for data augmentation and weak supervised learning via the following paragraph:
+For instance, we provided six more references and an example for data augmentation and weak supervised learning via the following paragraph:
```

Comment:
Sounds good. There could always be more references to add, but looking over the texts, I honestly think it works fine without additional references. Many of the things we mention are common knowledge, and there are no specific studies that exist for those, or it would be somewhere buried as a side note in some other study and hard to dig out.